v3/plugins/broadcommfd/cobol-language-support/0.8.3/meta.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+publisher: broadcommfd
+name: cobol-language-support
+version: 0.8.3
+type: VS Code extension
+displayName: COBOL Language Support
+title: COBOL Language Support
+description: COBOL Language Support standardizes the communication between language tooling and your code editor using the Language Server Protocol (LSP)
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/eclipse/che-che4z-lsp-for-cobol
+category: Language
+spec:
+  containers:
+    - image: eclipse/che-remote-plugin-runner-java8:7.3.0
```

Comment:
For non-release branches of Che, plugins should use the `next` tag of plugin runners. `7.3.0` is already 3 weeks old, and `7.4.0` has been released.

Reply:
We would prefer to release with version we built with and tested against. We experienced some issues during testing if we did not provide this version tag. Of course if the `next` tag is necessary, we can retest.

Reply:
Fair enough -- makes sense to me.